Automattic/wp-calypso
# Add benefits to cart for Jetpack AI

## Proposed Changes

* This diff adds items in the "Included with your purchase" section on the checkout page for Jetpack AI

## Testing Instructions

* Apply this patch locally
* Start your local calypso instance
* Navigate to http://calypso.localhost:3000/checkout/jetpack/jetpack_ai_monthly
* Confirm that the list of product features shows in the "Included with your purchase" section on the right-hand side of the checkout page

![Screenshot 2023-06-22 at 5 34 38 PM](https://github.com/Automattic/wp-calypso/assets/18016357/ef34f537-33c5-496b-ace2-bc558bd35d20)


## Pre-merge Checklist
